### DISCUSSION TITLE
Who's hiring? (May 2022)

### DISCUSSION THREAD

**Discussion by Nutlope** [vercel/next.js#36738] [Ideas]
# What is this?
New month, new post! This discussion is for companies looking for people with Next.js experience. Both permanent positions and contract work are welcome.

# Companies
Please use the following format followed by a brief description of the role:

> COMPANY_NAME | ROLE | LOCATION | Full-time | $100-150k base | LINK

Please only post if you personally are part of the hiring company—no recruiting firms or job boards. Only one post per company.

Good luck with recruiting!

## Sample Job Post

Here's a good example job post.

> Vercel | Senior Full-Stack Engineer | Remote (US, Canada, Europe, or Asia) | Full-time | $100-225k base | [Link](https://vercel.com/careers/senior-full-stack-engineer-amer)
>
> Vercel’s hiring across the board on various engineering teams and positions, but the above link is for our full-stack engineering position. Vercel is innovating on and for the web and at the edge. We build tools and workflows developers love and deliver outstanding performance for end users.
>
> I've been at the company for 2 years and still find plenty of opportunity for impact, and love the teammates I'm working with. Feel free to apply using the link above or reach out to me directly.

---

*If you're looking for last month's discussion, see #35840. At least one company has [hired someone](https://twitter.com/Nutlope/status/1475929984174895105) as a result of it!* 

If you hire someone through this job board, please reach out to me (hassan@vercel.com) so I can send you some swag :)

If you're a developer looking to post about themselves, use #36737.

**Comment by danielsdesk** [comment]
ocV!BE | Sr Product Engineer - UI/Web | Remote (US/CAN) | Full-time | Permanent

Hi, we're hiring for a Senior Software Product Engineer UI/Web. React.js/Next.js and more is applicable experience.

[Apply Here](https://anaheimducks.wd5.myworkdayjobs.com/en-US/HC/job/Senior-Software-Product-Engineer--UI-Web_R-2022-183-2)

The Work:  As a UI/Web Senior Software Product Engineer, you will also work with a multidisciplinary team such as System Engineers, Backend Engineers, and Creative Technologists—to help guide product visions to reality by:

- Being responsible for designing and implementing intelligent and efficient UIs, spinning up and maintaining robust UI components, and developing speedy and modular content management systems by developing and delivering impactful code that enables the success of Litehouse user applications and embedded webviews
- Implementing modern web application stacks as well as executing user interface designs for a variety of product applications, experiential applications, and UI tools in potentially non-traditional ways
- Participating in prototyping, testing, developing, deploying, maintaining, and evolving items on the Litehouse product roadmap
- Contributing to exceptional software, hardware, and design solutions that exceed expectations
- Deeply understanding our product in support of its development, growth, and enablement of user success
- Contributing to exceptional software, hardware, and design solutions that exceed expectations
- Writing clean, end-to-end code in support of the above
- Understanding the bigger picture and the purpose of software to ensure and improve its quality and fitness-for-purpose


Skills

- Strong understanding of Modern UI architectures, React, and JavaScript, with some experience with Next.js and Webpack
- Responsive UI Development w/ HTML, CSS, and related web technologies
- Expertise in integrating with APIs (REST and GraphQL w/ Relay) and Socket communication (Websocket/STOMP)
- Expertise in advanced UI State Management techniques for React with libraries such as Redux, XState, and Jotai
- Ability to participate within Agile Component-Driven Development and UI Review/Prototyping processes
- Someone who thrives in novel, first-of-a-kind settings, with creative contribution informed by technical acumen
- An excellent independent problem solver and proactive communicator who can self-direct
- Ability to learn and master new technologies very quickly


Culture & Fit

This role is an excellent fit for anyone who:

- Is an independent, careful thinker who is passionate about the work and places a premium on craftsmanship, professional responsibility, data privacy, and ethical design
- Is excited to work on bleeding-edge technologies and devote mindshare to solving the problems of tomorrow, today
- Has interest in microfrontends via Module Federation, and UI Review with Storybook
- Has interest in Adaptive/Intelligent UIs and workflows
- Has interest in 3D technologies such as WebGL/Three.js/React-Three-Fiber for AR/VR/XR, and/or gaming, as well as other audio/video media technologies such as music production or event production
- Has interest in WebAssembly for Web, Desktop, and Mobile Apps, and potential languages for it such as Rust/Assemblyscript
- Has interest in creative technology, UX, workflow management, systems thinking, art, architecture, smart cities, and AI
- A healthy appreciation for and use of Slack’s Giphy integration
- Lives in or might be interested in relocating to sunny Southern California

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job board for June is up (https://github.com/vercel/next.js/discussions/37379) and you're more than welcome to repost this message there!

**Comment by joshma** [comment]
Airplane | Frontend Engineer | Remote (US timezones) | Full-time | $150-200k base | [Link](https://www.airplane.dev/careers?ashby_jid=6010d440-dfdc-4c4e-8361-9e25a0c658b8)

Hi all! I'm Josh, one of the founders at [Airplane](https://www.airplane.dev). We build dev infrastructure for internal tools.

Airplane is building "legos" for developers to rapidly assemble into internal admin panels, migration tools, support utilities, and more. As a frontend engineer, you would build the interfaces that hundreds of teams use to power the internals of their business.

If you're interested, please reach out to me (josh@airplane.dev) or use the link above!

**Reply by Tirupatibalaji-Dev** [reply]
I would love to apply.

**Reply by arvindk427** [reply]
Interested

**Reply by aymanesarrar** [reply]
Interested

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job board for June is up (https://github.com/vercel/next.js/discussions/37379) and you're more than welcome to repost this message there!

[a comment by an automated account is not shown]

**Comment by jserrao** [comment]


**Reply by arvindk427** [reply]
Interested

**Reply by aymanesarrar** [reply]
interested

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job board for June is up (https://github.com/vercel/next.js/discussions/37379) and you're more than welcome to repost this message there!

**Reply by jserrao** [reply]
Thanks @Nutlope - this was really helpful. We may be back in the future...

**Comment by omure** [comment]
Omure | Senior React.js Engineer | Remote (Europe, Americas) | Full-time | [Link](http://omure.com/jobs/senior-react)

We're looking for experienced React/Next devs to join our Front-end team.
There's no legacy or client work involved (we incubate startups) and hours are flexible.

Currently we offer a lot of room to grow fast - within the next year.

**Reply by definoob** [reply]
This company never sends reply. Was expecting decline message at least.

**Reply by omure** [reply]
> This company never sends reply. Was expecting decline message at least.

Hey @definoob  front-end positions are always open and we're still interviewing people daily.

As far as i know, HR keeps everyone posted after the initial interviews.
If that wasn't the case, please provide the details via email to me ( nikola@omure.com ) as I'd like to investigate.
Thank you.

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job board for June is up (https://github.com/vercel/next.js/discussions/37379) and you're more than welcome to repost this message there!

**Comment by GregDoyleRS** [comment]
Deutsche Bank | Full Stack Engineer | Cary, NC (USA - Hybrid) | Permanent | [Link](https://careers.db.com/professionals/search-roles/#/professional/job/19374) 


Your Skills and Experience:

Extensive experience in IT industry with hands-on development in Programming in one or more of the following languages / frameworks- Java, NodeJS, Kotlin, ReactJS, Angular and/or Oracle tools
Experience with Service Orientated Architecture or Microservices
Build platform enabler architectural use cases
Experience with Test Driven Development (TDD) and Behavior Driven Development (BDD) including testing frameworks (Junit, Cucumber and/or Selenium
Expertise in Bit Bucket, JIRA and ability of using Confluence

**Reply by arvindk427** [reply]
Interested

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job board for June is up (https://github.com/vercel/next.js/discussions/37379) and you're more than welcome to repost this message there!

**Comment by fofiu** [comment]
Monogram | **Front End Developer** | Remote (Americas Preferred) | Full-time | [Link](https://monogram.io/careers)

## Introduction
You don’t work at Monogram to remove roadblocks, you work here to pave new paths. You don’t just solve problems, you join a family of designers & developers who collaborate like clockwork, rise together, fall together, and do their life’s work here.

A front-end developer at Monogram is someone that is motivated to combine the art of design with the art of programming. An ideal front-end developer is a designer first, but a HTML and CSS wizard 🧙🏻 at the same time.

## Duties

- Collaborate on a daily basis with a team of product managers, engineers, and UI/UX designers to understand business requirements, user experience goals and pain points
- 🔥 Meticulously translate the UI/UX design wireframes to actual code mobile first
- Develop new user-facing features
- Build reusable components and libraries for future use
- Optimize application for maximum speed and scalability
- Ensure the technical feasibility of UI/UX designs

## Skills
### 🚨 Required
- Proficient in HTML5
- Solid understanding of CSS (including preprocessors like SASS)
- Tailwind CSS
- Next.js/React

### 🧁 Bonus
- TypeScript
- Knowledge of JAMstack
- Svelte
- Figma
- Three.js


### 🤔 Interested?
Send portfolio & résumé to careers@monogram.io

**Reply by aymanesarrar** [reply]
interested

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job board for June is up (https://github.com/vercel/next.js/discussions/37379) and you're more than welcome to repost this message there!

**Comment by hayanisaid** [comment]
Genesis Financial Solutions | Senior Mobile Developer | Remote | Full-time  | [link](https://careers-genesis-fs.icims.com/jobs/2195/sr.-mobile-developer/job)

My team is hiring a Senior Mobile Developer with React Native experience to join our team in Oregon. Our company values diversity and has a strong team culture, and we offer a range of great benefits including the option to work remotely. As a Senior Mobile Developer, you will have the opportunity to work on a variety of exciting projects and have a real impact on the direction of our mobile development efforts. We are looking for someone with strong problem-solving skills and the ability to work well in a team environment. If you have a passion for mobile development and are excited about the opportunity to join a dynamic and fast-paced company, we encourage you to apply for this position. 
Learn more about the position here: https://careers-genesis-fs.icims.com/jobs/2195/sr.-mobile-developer/job
- I'd be happy to answer any question you may have
